### ISSUE TITLE
Understandable pre-set sensor measurement configurations 

### ISSUE THREAD

**Issue by antoniodini** [macmade/Hot#87]
This app measures an incredible amount of sensors indicating the most diverse temperatures with fluctuations of even more than 20 degrees. 
Since the individual sensors in Apple Silicon have labels that are not understandable, I think it would be useful to have an option of a simplified choice mode. 
A mode with three-four "macro" options (i.e.: "Basic", "Cpu perfornance", "Memory performance", "Ssd performance") in which only the relevant sensors are measured. 
Otherwise in my opinion it is not clear what the temperature of the Mac actually is.

**Comment by macmade** [comment]
Would be great indeed.
Only problem is there's no information about the sensors. Only their names are available.

**Comment by dmukhin** [comment]
Let me add few coins, I think it fits to issue topic some way:  at least on my Air M1 by default in Hot Preferences > Select Sensors nothing selected and "Include Only Selected sensors" choosen. Thus it is not clear which sensor values read to calculate current temp shown in bar (and detached graph). Maybe better to make it clear somehow. And BTW could you pls tell me which sensors used for my config? Great thnks!

**Comment by macmade** [comment]
As I said, there’s no information about which sensors are available on a specific machine.

**Comment by dmukhin** [comment]
> there’s no information about which sensors are available on a specific machine

Yep, that's clear. But how does Hot calculate temperature shown in bar? Maybe it's misunderstanding from my side, when no sensors selected in preferences, does Hot use some system API (and doesn't read SMB or IOHID) to get temp vaue displayed as current?

**Comment by dmukhin** [comment]
> when no sensors selected in preferences

Maybe I got it (sorry, if I'm wrong, really didn't check carefully, just saw https://github.com/macmade/Hot/blob/9405e2a9d7e5d07c50f5e17be81eaa290f2c442a/Hot/Classes/ThermalLog.swift#L73 and https://github.com/macmade/Hot/blob/9405e2a9d7e5d07c50f5e17be81eaa290f2c442a/Hot/Classes/ThermalLog.swift#L135 etc). 
In case no sensors selected in preferences it uses pACC* and eACC* sensors, right?

**Comment by macmade** [comment]
pACC and eACC are sensors found only on the MacBook Air M1.
